raveandroid/src/main/java/com/flutterwave/raveandroid/card/CardFragment.java

```diff
@@ -850,7 +855,6 @@ public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
         public void afterTextChanged(Editable editable) {
             String input = editable.toString();
             SimpleDateFormat simpleDateFormat = new SimpleDateFormat("MM/yy");
```

Comment:
You're creating an instance of `simpleDateFormat` each time the text changes which is expensive. Try to create it once in the `ExpiryWatcher()`'s constructor and use that instance in `line 860` instead just like you did for your `calendar` variable.

Reply:
Oh true. Correcting that...

Reply:
DONE.